cmd/restic/cmd_copy.go

```diff
@@ -52,6 +53,7 @@ func init() {
 	f.StringArrayVarP(&copyOptions.Hosts, "host", "H", nil, "only consider snapshots for this `host`, when no snapshot ID is given (can be specified multiple times)")
 	f.Var(&copyOptions.Tags, "tag", "only consider snapshots which include this `taglist`, when no snapshot ID is given")
 	f.StringArrayVar(&copyOptions.Paths, "path", nil, "only consider snapshots which include this (absolute) `path`, when no snapshot ID is given")
+	f.IntVar(&copyOptions.CopyWorkers, "copy-workers", 8, "number of concurrent workers for copying blobs")
```

Comment:
Please remove the `--copy-workers` option, it won't be relevant any more one #3513 is merged. Just create a constant that hardcodes 8 workers.